ui/components/multichain/asset-picker-amount/asset-picker-modal/asset-picker-modal-network.tsx

```diff
@@ -221,63 +235,151 @@ export const AssetPickerModalNetwork = ({
         <Box
           className="multichain-asset-picker__network-list"
           display={Display.Flex}
+          flexDirection={FlexDirection.Column}
         >
+          {testNetworks.length > 0 ? (
+            <Box padding={4}>
+              <Text
+                variant={TextVariant.bodyMd}
+                color={TextColor.textAlternative}
+              >
+                {t('enabledNetworks')}
+              </Text>
+            </Box>
```

Comment:
nit: I couldn't see in the recording where this appears but could you reduce some html bloat here by applying the padding directly to the `Text` component?

```suggestion
            <Text
              variant={TextVariant.bodyMd}
              color={TextColor.textAlternative}
              padding={4}
            >
              {t('enabledNetworks')}
            </Text>
```

Reply:
Updated here https://github.com/MetaMask/metamask-extension/pull/30959/commits/e91d2df6f545c7b019543d5241c65afe7786559b